### DISCUSSION TITLE
How to view only unmatched/untagged scenes?

### DISCUSSION THREAD

**Discussion by keithghrobertson** [stashapp/stash#2917] [Q&A] [answered]
As the title asks - how do I view only my unmatched scenes? It seems when I go to the tagger and scan, it'll show items I've already tagged. Is there a way to show only unmatched/untagged scenes?

**Comment by 7dJx1qP** [comment, marked as answer]
Add a filter for stash id is null
![image](https://user-images.githubusercontent.com/38586902/190544332-3abc9fd7-e1a6-4782-9f12-acb67b932864.png)
